src/modules/validator.js

```diff
@@ -0,0 +1,96 @@
+class Validator {
+
+    constructor(chluIpfs) {
+        this.chluIpfs = chluIpfs;
+    }
+
+    async validateReviewRecord(reviewRecord, validations = {}) {
+        const rr = cloneReviewRecord(reviewRecord);
+        const v = Object.assign({
+            validateVersion: true,
+            validateMultihash: true,
+            validateHistory: true
+        }, validations);
+        if (v.validateVersion) this.validateVersion(rr);
+        if (v.validateMultihash) await this.validateMultihash(rr, rr.hash.slice(0));
+        if (v.validateHistory) await this.validateHistory(rr);
```

Comment:
Should we add a placeholder function called `validateReviewSignature` that always returns true and marked with a TODO